e2e_test/sink/kafka/create_sink.slt

```diff
@@ -97,6 +97,17 @@ create sink si_kafka_upsert from t_kafka with (
     primary_key = 'id',
 );
 
+statement ok
+create sink si_kafka_upsert_schema from t_kafka with (
+    connector = 'kafka',
+    properties.bootstrap.server = '127.0.0.1:29092',
+    topic = 'test-rw-sink-upsert-schema',
+    type = 'upsert',
```

Comment:
`type = 'upsert'` is deprecated in favor of `format upsert` below. We do not need both